lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/IssueCredentialHandler.java

```diff
@@ -109,11 +109,11 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             SignedJWT signedJWT =
                     verifiableCredentialService.generateSignedVerifiableCredentialJwt(
-                            sessionItem.getSubject(), personIdentity, kbvItem);
+                            sessionItem.getSubject(), personIdentity, kbvItem, sessionItem);
```

Comment:
just pass in sessionItem

Reply:
`this.sessionService.getSessionByAccessToken(accessToken);`https://github.com/govuk-one-login/ipv-cri-lib/blob/main/src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java#L121 would throw an error of there is no sessionItem 

so don't pass the entire sessionItem instead pass `sessionItem.getEvidenceRequest` which is what is needed on line 112